Разметка всех страниц

AlexWelm opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2252263/tasks/5)

AlexWelm committed:
Добавляет файл страницы каталога

AlexWelm committed:
Переименовывает файл каталога

AlexWelm committed:
Размечает header

AlexWelm committed:
Добавляет атрибут адресса ссылок выпадающего меню

AlexWelm committed:
Размечает секцию каталога

AlexWelm committed:
Размечает секцию промо

AlexWelm committed:
Изменяет сылки в секции каталога

AlexWelm committed:
Добавляет секции сервиса, доставки, о нас, контакт

AlexWelm committed:
Добавляет разметку секции подписки

AlexWelm committed:
Добавляет разметку footer главной страницы

AlexWelm committed:
Копирует в каталог header и footer с главной

AlexWelm committed:
Добавляет ссылку в навигацию каталога - Выйти

AlexWelm committed:
Оборачивает логотип в ссылку в header и footer

AlexWelm committed:
Добавляет разметку main страницы каталога

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+          </li>
+          <li class="navigation-user-item">
+            <a class="navigation-link" href="#"><span class="visualliy-hidden">Корзина</span></a>
+          </li>
```
этим элементам "visually-hidden" не нужен. В ссылках должен быть текст как на макете.
Рядом с корзиной нужен спан с кол-вом товаров.

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+            </button>
+            <button class="promo-slider-button promo-slider-button-next" type="button">
+              <span class="visually-hidden">Следующий товар</span>
+            </button>
```
Кнопки "next" "prev" лучше  не повторять в каждом слайде, а вынести в отдельный блок.

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+      <section class="service">
+        <h2 class="service-title">Сервис</h2>
+        <ol class="service-slider-list">
+          <li class="service-slaider-item">
```
slaider - опечатка в классе

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+          </li>
+        </ol>
+      </section>
+      <article class="delivery">
+        <h2 class="visually-hidden">Доставка под заказ</h2>
+        <img src="" alt="Иконка грузовика">
+        <p>Доставим редкий товар под заказ</p>
+        <button class="delivery-button">
+          <span class="visually-hidden">Подробнее</span>
+        </button>
+      </article>
```
Весь блок должен быть ссылкой (смотри стайлгайд). А иконку машинки лучше сделать декоративным изображением

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+          <a class="contacts-link" href="#">Напишите нам</a>
+        </ul>
+      </section>
+      <section class="subscription">
+       <h2 class="visually-hidden">Подписка на рассылку</h2>
+       <p class="subscription-title">Подпишитесь на рассылку</p>
```
"Подпишитесь на рассылку" из макета можно сделать заголовком, а скрытый заголовок убрать. Но это скорее совет и не обязательно к исправлению.

vindimm commented on index.html:
```diff
@@ -5,8 +5,309 @@
+       <p class="subscription-title">Подпишитесь на рассылку</p>
+       <p class="subscription-text">Еженедельный дайджест с обзорами новинок, длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.</p>
+       <form action="https://echo.htmlacademy.ru/courses" method="post">
+        <input class="subscription-input" type="email" name="email-subscription" placeholder="Ваш e-mail">
```
input required

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,246 @@
+              </li>
+            </ul>
+          </fieldset>
+          <button class="button-catalog" type="button">Показать</button>
```
нужно `type="submit"`

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,246 @@
+          <option value="new">Новые</option>
+          <option value="all">Все</option>
+        </select>
+        <button class="button-sorting-up">
+          <span class="visually-hidden">Пролистать вверх</span>
+        </button>
+        <button class="button-sorting-down">
+          <span class="visually-hidden">Пролистать вниз</span>
+        </button>
```
Это кнопки "по возрастанию" "по убыванию". Для страниц у нас пагинация ниже

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,246 @@
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link" href="#">2</a>
+          </li>
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link" href="#">3</a>
+          </li>
+        </ol>
+        <p class="product-card-pagination-text">...</p>
+        <ol class="product-card-pagination-list" start="15">
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link" href="#">15</a>
+          </li>
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link pagination-next" href="#">Вперед</a>
+          </li>
+        </ol>
```
Лучше сделать пагинацию одним списком `ul` с ссылками. Элемент "точки..." можно сделать спаном.

vindimm reviewed (changes requested):
В целом очень хороший код. Понравилось использование `dl` `dt` `dd`.
Я бы иконки в блоке "Доставка, гарантия..." сделал не контентными а декоративными, но это спорный вопрос.
В футере можно некоторые section заменить на простые div. Соцсети точно оставить section, а остальное на твой вкус.

Вноси эти небольшие правки и ДЗ готово.

AlexWelm committed:
Исправляет ссылки в навигации

AlexWelm committed:
Выносит next и prev в отдельный блок

AlexWelm committed:
Update index.html

Выносит next и prev  в отдельный блок

AlexWelm committed:
Исправление опечаток в секции - service

AlexWelm committed:
Update index.html

Переносит код из article в отдельную ссылку, убирает скрытый заголовок в subscription

AlexWelm committed:
Добавляет атрибут required полю адреса email

AlexWelm committed:
Заменяет атрибут у кнопки отправки формы на submit

AlexWelm committed:
Меняет название кнопок в сортировке

AlexWelm committed:
Делает пагинацию одним списком

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,248 @@
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link" href="#">3</a>
+          </li>
+          <li class="product-card-pagination-item">
+            <a class="product-card-pagination-link" href="#">
+              <span class="product-card-pagination-text">...</span>
+            </a>
+          </li>
```
Точки не ведут на какую-то страницу. Лучше убрать ссылку или убрать `href`

vindimm commented on catalog.html:
```diff
@@ -0,0 +1,248 @@
+          <li class="navigation-user-item">
+            <a class="navigation-link" href="#">Сравнить</a>
+          </li>
+          <li class="navigation-user-item">
+            <a class="navigation-link" href="#"><span class="visualliy-hidden">Количество товара в корзине</span> Корзина</a>
+          </li>
```
Нужно добавить контейнер с числом `2`. В макете оно есть, а у тебя - нет.
